### PR TITLE
CreatAt and UpdateAt Added (Entity, IRepository Repository)

### PR DESCRIPTION
**Pull request by Ellyria34**

### CODE REVIEW

File: QuikForm.Entities/Record.cs [hunk 1]
```diff
@@ -11,6 +11,10 @@ public class Record
     [Key]
     public int Id { get; set; }
 
+    public DateTime CreateAt { get; set; }
+
+    public DateTime UpdateAt { get; set; }
+
```
Comment (htkisui) [inline QuikForm.Entities/Record.cs:19]: Both [Required]

File: QuikForm.Repositories/Repositories/RecordRepository.cs [hunk 2]
```diff
@@ -21,6 +21,7 @@ public RecordRepository(ApplicationDbContext context)
 
     public async Task CreateAsync(Record record)
     {
+        record.CreateAt = DateTime.Now;
```
Comment (htkisui) [inline QuikForm.Repositories/Repositories/RecordRepository.cs:24]: with UpdatedAt = DateTime.Now;

File: QuikForm.Repositories/Repositories/RecordRepository.cs [hunk 3]
```diff
@@ -42,4 +43,15 @@ public async Task<Record> GetByIdAsync(int id)
         Record record = await _context.Records.FirstOrDefaultAsync(r => r.Id == id) ?? throw new RecordNotFoundException();
         return record;
     }
+
+    public async Task<Record> UpdateAsync(Record record)
+    {
+        Record recordToUpdate = await _context.Records.FirstOrDefaultAsync(r => r.Id == record.Id) ?? throw new();
```
Comment (htkisui) [inline QuikForm.Repositories/Repositories/RecordRepository.cs]: new RecordNotFoundException();

File: QuikForm.Entities/Record.cs (hunk 1 above)
Comment (Ellyria34) [reply QuikForm.Entities/Record.cs:19]: done

File: QuikForm.Repositories/Repositories/RecordRepository.cs (hunk 3 above)
Comment (Ellyria34) [reply QuikForm.Repositories/Repositories/RecordRepository.cs]: done

File: QuikForm.Repositories/Repositories/RecordRepository.cs (hunk 2 above)
Comment (Ellyria34) [reply QuikForm.Repositories/Repositories/RecordRepository.cs:24]: done

**Review by htkisui — APPROVED** [review]

**Pull request merged**